.github/workflows/themes-weekly-pull.yml

```
@@ -0,0 +1,46 @@
+name: themes-weekly-pull
+
+on:
+  schedule:
+    # run every friday at 5AM
+    - cron: '0 5 * * 5'
+

```

```suggestion
  workflow_dispatch:

```

This would allow the workflow to be triggered manually. My idea is it might be useful to have the option to pull in theme changes on demand in addition to the scheduled trigger.

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch